### ISSUE TITLE
Encountered zero divided

### ISSUE THREAD

**Issue by tartcuru** [hkaneko1985/dcekit#1]
分散0列が入っているdatasetの場合、下記の箇所で0割発生→nan変換→モデルが構築できなくなります。
コール元で分散0列を削除することも可能ですが、train_test_splitの結果次第で分散0列が発生することもあるので、dcekit内で対処されてあるとうれしいです。

dcekit\generative_model\gmr.py Line 344 in cv_opt.
`autoscaled_dataset = (dataset - dataset.mean(axi=0)) / dataset.std(axis=0, ddof=1)`
